transform/transformers/MWSSTransformer.py

```diff
@@ -437,7 +437,7 @@ def pack(self, img_seq=None):
             doc.build(PDFTransformer.get_elements(survey, self.response))
 
             # Create page images from PDF
-            img_tfr = ImageTransformer(self.log, survey, self.response)
+            img_tfr = ImageTransformer(self.log, survey, self.response, self.ids.seq_nr)
```

Comment:
Happy to approve once we show that defaulting to 0 in `seq_nr` doesn't cause further issues downstream via unit tests